packages/components/src/Combobox/hooks/useComboboxAccessibility.ts

```diff
@@ -25,29 +31,34 @@ export function useComboboxAccessibility(
   useRefocusOnActivator(open);
 
   const popperRef = useFocusTrap<HTMLDivElement>(open);
-  const {
-    styles: popperStyles,
-    attributes,
-    update,
-  } = usePopper(wrapperRef.current, popperRef.current, {
-    modifiers: [
-      {
-        name: "flip",
-        options: {
-          fallbackPlacements: ["top-start"],
-        },
-      },
+
+  const { floatingStyles, update, context } = useFloating({
+    elements: {
+      reference: wrapperRef.current,
+      floating: popperRef.current,
+    },
+    open,
+    onOpenChange: openState => {
+      if (!openState) handleClose();
+    },
+    middleware: [
+      autoPlacement({ allowedPlacements: ["bottom-start", "top-start"] }),
```

Comment:
It's great that we can now utilize `autoPlacement` - I'm just wondering if we need to target the styling a bit more considering the gap we have when the menu is opened below the activator? 

![Screenshot 2025-03-25 at 2 20 38 PM](https://github.com/user-attachments/assets/1b353080-df08-4e47-92a2-5a5e89a6cbab)

Reply:
Resolved in 0c08c33bf